### PR TITLE
ref(py3): Be consistent with float / int in action_handlers

### PR DESCRIPTION
**Pull request by evanpurkhiser**

### CODE REVIEW

File: src/sentry/incidents/action_handlers.py [hunk 1]
```diff
@@ -154,18 +154,20 @@ def format_duration(minutes):
...
+        return "{:d} hour{}".format(hours, pluralize(hours))
 
     if minutes >= 1:
-        return "{} minute{}".format(minutes, pluralize(minutes))
+        minutes = int(minutes)
+        return "{:d} minute{}".format(minutes, pluralize(minutes))
 
-    seconds = minutes // 60
-    return "{} second{}".format(seconds, pluralize(seconds))
+    seconds = int(minutes // 60)
```
Comment (EvanPurkhiser) [inline src/sentry/incidents/action_handlers.py:169]: This is wrong, I'll fix it later

**Review by wedamija — APPROVED** [review]
lgtm, probably worth fixing the seconds bug before merging.

File: src/sentry/incidents/action_handlers.py [hunk 2]
```diff
@@ -154,18 +154,20 @@ def format_duration(minutes):
...
-        days = minutes // 1440
-        return "{} day{}".format(days, pluralize(days))
+        days = int(minutes // 1440)
+        return "{:d} day{}".format(days, pluralize(days))
 
     if minutes >= 60:
-        hours = minutes // 60
-        return "{} hour{}".format(hours, pluralize(hours))
+        hours = int(minutes // 60)
+        print(hours)
```
Comment (EvanPurkhiser) [inline src/sentry/incidents/action_handlers.py:162]: lol maybe without printing

File: src/sentry/incidents/action_handlers.py [hunk 3]
```diff
@@ -154,18 +154,19 @@ def format_duration(minutes):
     """
 
     if minutes >= 1440:
-        days = minutes // 1440
-        return "{} day{}".format(days, pluralize(days))
+        days = int(minutes // 1440)
```
Comment (BYK) [inline src/sentry/incidents/action_handlers.py:157]:
```
>>> type(3 // 2)
<type 'int'>
```

This is the case for both in Py2 and Py3 so why to we need the cast?

**Review by BYK — COMMENTED** [review]
Sorry, I don't understand why we need these changes. Can you explain a bit?

File: src/sentry/incidents/action_handlers.py (hunk 3 above)
Comment (BYK) [reply src/sentry/incidents/action_handlers.py:157]: Oh wait, because `minutes` can be a `float`?

Comment (EvanPurkhiser) [reply src/sentry/incidents/action_handlers.py:157]: Yeah

Comment (BYK) [reply src/sentry/incidents/action_handlers.py:157]: Then wouldn't it be safer/easier to do `minutes = int(minutes)` at the top of the function and leave the remaining as they are?

**Pull request merged**